pyquil/api/_quantum_computer.py

```diff
@@ -249,8 +249,7 @@ def list_quantum_computers(connection: ForestConnection = None,
 
     qc_names: List[str] = []
     if qpus:
-        # TODO: add deployed QPUs from web endpoint
-        pass
+        qc_names += list(list_lattices().keys())
 
     if qvms:
         qc_names += ['9q-generic-qvm', '9q-generic-noisy-qvm']
```

Comment:
"9q-generic" -> "9q-square" (I missed this in the great square-lattice-refactor of #571